bench/php/optimized/var.ll

```diff
@@ -6486,64 +6486,57 @@ php_var_serialize_destroy.exit:                   ; preds = %30, %.thread.i, %33
 
 46:                                               ; preds = %41
   call void @_efree(ptr noundef nonnull %35) #14
-  %.pre.pre = load ptr, ptr getelementptr inbounds nuw (i8, ptr @executor_globals, i64 960), align 8, !tbaa !79
-  %47 = icmp ne ptr %.pre.pre, null
   br label %smart_str_free_ex.exit
 
-smart_str_free_ex.exit:                           ; preds = %37, %41, %46, %36
-  %48 = phi i1 [ true, %36 ], [ %47, %46 ], [ true, %41 ], [ true, %37 ]
-  call void @llvm.assume(i1 %48)
```

Comment:
Side note: Should probably avoid sinking assumes if it makes the assume condition a phi node...